cpu/cpupower_monitor.py

```diff
@@ -41,7 +41,14 @@ def setUp(self):
             if 'Available idle states: ' in line:
                 self.states_list = (line.split('Available idle states: ')[-1]).split()
                 break
-        self.log.info("Idle states on the system are: ", self.states_list)
+        self.log.info("Idle states on the system are: %s", self.states_list)
+
+        for line in output.splitlines():
+            if 'Number of idle states: ' in line:
+                self.states_tot = int(line.split('Number of idle states: ')[1])
+                break
+        self.log.info("Total Idle states: %d", self.states_tot)
+
```

Comment:
do not use white space in same block of code

Reply:
Done